spec/minimal_repo.rb

```diff
@@ -5,13 +5,7 @@ module MinimalRepo
   class << self
     DIRECTORIES = [
       '.git',
-      'config',
-      'hosts',
-      'genders',
-      'dhcp',
       'files',
```

Comment:
Do we still need to have this array if it only has `.git` and `files`? Can this be done the same way as `config`, `hosts`, etc?